Drop-In/src/main/java/com/braintreepayments/api/SupportedPaymentMethodsFragment.java

```diff
@@ -183,7 +181,7 @@ private void showVaultedPaymentMethods(List<PaymentMethodNonce> paymentMethodNon
 
             mVaultedPaymentMethodsView.setAdapter(vaultedPaymentMethodsAdapter);
 
-            if (dropInRequest.isVaultManagerEnabled()) {
+            if (dropInRequest.getEnableVaultManager()) {
```

Comment:
I kind of like the `is` convention for booleans. Was this change necessary because of Kotlin conversion?

Reply:
Ah ok I was mistaken with the Kotlin conversion. It looks like I can change this to `setVaultManagerEnabled` and `isVaultManagerEnabled` as the getter. Then in Kotlin it is set like `dropInRequest.isVaultManagerEnabled = true`. I think that looks better I'm going to take a second pass at all the boolean setter/getter names.